op-e2e/actions/helpers/l2_sequencer.go

```diff
@@ -30,11 +30,11 @@ type MockL1OriginSelector struct {
 	originOverride eth.L1BlockRef // override which origin gets picked
 }
 
-func (m *MockL1OriginSelector) FindL1Origin(ctx context.Context, l2Head eth.L2BlockRef) (eth.L1BlockRef, error) {
+func (m *MockL1OriginSelector) FindL1Origin(ctx context.Context, l2Head eth.L2BlockRef, recoverMode bool) (eth.L1BlockRef, error) {
 	if m.originOverride != (eth.L1BlockRef{}) {
 		return m.originOverride, nil
 	}
-	return m.actual.FindL1Origin(ctx, l2Head)
+	return m.actual.FindL1Origin(ctx, l2Head, recoverMode)
```

Comment:
From conversation with @sebastianst: we may be able to avoid threading this bool through a lot of function signatures by adding it to the sequencer's state.

Reply:
Tried this out here [70be0ba](https://github.com/ethereum-optimism/optimism/pull/14823/commits/70be0ba0d37a2e71b735dee827e0d1d8e24c54ce)